feature-libs/cart/import-export/core/config/import-export-config.ts

```diff
@@ -1,17 +1,12 @@
 import { Injectable } from '@angular/core';
 import { Config } from '@spartacus/core';
-
-export type FileValidity = {
-  // size unit is MB
-  maxSize?: Number;
-  allowedExtensions?: string[];
-  checkEmptyFile?: Boolean;
-};
+import { FileValidity } from '../model/import-export.model';
 
 @Injectable({
   providedIn: 'root',
   useExisting: Config,
 })
-export abstract class FileValidityConfig {
+export abstract class ImportExportConfig {
   fileValidity: FileValidity;
+  file: { separator: string };
```

Comment:
We'd like not to pollute the root level of the config with too many properties, but rather wrap configs of each feature inside one feature-lib specific key.

For example

```ts
class ImportExportConfig {
  importExport: { 
     fileValidity: ...
     file: ...
  }
}
```